tests/test_triton_flashattention.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
```

Comment:
@fmassa I've added the op to `test_mem_eff_attention.py`, is there anywhere else in the file it should be added? 

@danthe3rd I thought it would be nice to have a simple test separate from the op logic, for easier debugging. But I can remove it if it seems like overkill.

Reply:
Actually, I ended up deleting the test, since I'll just use the implementation directly.